platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheIntegrationTest.groovy

```diff
@@ -409,4 +409,19 @@ class ConfigurationCacheIntegrationTest extends AbstractConfigurationCacheIntegr
         configurationCache.assertNoConfigurationCache()
         outputContains("isConfigurationCacheRequested=true")
     }
+
+    def "internal load-after-store flag is deprecated"() {
+        def configurationCache = newConfigurationCacheFixture()
+
+        when:
+        executer.expectDeprecationWarning("The org.gradle.configuration-cache.internal.load-after-store system property has been deprecated. Starting with Gradle 9.0, it will not be possible to disable load-after-store behavior of Configuration Cache. The behavior is enabled by default. Avoid using the internal flag.")
```

Comment:
💅 Breaking this long line into several with `+`, e.g. at sentence bounds, may improve readability slightly

Reply:
Done